### PR TITLE
Zero-Knowledge MPW upgrade

### PR DESCRIPTION
**Pull request by JSKitty**
## Abstract

Considering Zero-Knowledge (zk) is one of the most powerful privacy protocols, it's a perfect usecase for DIRECT wallet integration.

After merging this PR, MPW will be entirely zkSNARKs-based, there will be zero knowledge of MPW.

image

> **the most private wallet, is having no wallet.** - Unknown (Due to Zero Knowledge)

## What does this PR address?
It achieves maximum privacy with zero knowledge.

## What features or improvements were added?
Nothing, that's the point, no knowledge.

## How does this benefit users?
They gain zero knowledge.

---

<details>
  <summary>Zero-Knowledge Button - Do Not Click</summary>
  April Fools.  
</details>

### CODE REVIEW

**Review by panleone — APPROVED** [review]
tACK 100b3302781d32d70d566860543e44757cb0be77
finally the code is secure and private LGTM

File: locale/uwu/translation.js [hunk 1]
```diff
@@ -1,152 +0,0 @@
...
-    // Intro
-    title: "Wewcome to",                      //Welcome to
-    titleName: "My PIVX Wawwet!",                  //My PIVX Wallet!
-
-    cardOneTitle: "Be da bank!",               //Be your own Bank!
-    cardOneDesc: "MyPIVXWawwet has <b>nowo custody</b> over yowour funds. You are in full owonership of your keys and your PIV.",                //MyPIVXWallet has <b>no custody</b> over your funds. You are in full ownership of your keys and your PIV. 
-    cardOneLink: "Knowo mowore",                //Know more
-
-    cardTwoTitle: "Univewsal and Portabwe",               //Universal and Portable
-    cardTwoDesc: "Yowou can genewate cwyptogwaphicawwy-secuwe adwesses with yowour bwowoser and hawdware.",                //You can generate cryptographically-secure addresses with your browser and hardware.
```
Comment (Duddino) [inline locale/uwu/translation.js:36]: ​

File: assets/music.mp3
Comment (Duddino) [inline assets/music.mp3:1]: ​

File: package.json
Comment (Duddino) [inline package.json:1]: ​

File: scripts/settings.js [hunk 2]
```diff
@@ -1,394 +0,0 @@
-import {
-    doms,
-    getBalance,
-    getStakingBalance,
-    updateStakingRewardsGUI,
-} from './global.js';
-import { fWalletLoaded, masterKey } from './wallet.js';
-import { cChainParams } from './chain_params.js';
-import { setNetwork, ExplorerNetwork, getNetwork } from './network.js';
-import { createAlert } from './misc.js';
```
Comment (Duddino) [inline scripts/settings.js:10]: ​

File: scripts/prices.js [hunk 3]
```diff
@@ -1,82 +0,0 @@
...
-import { cMarket, fillCurrencySelect } from './settings';
-
-/**
- * CoinGecko's endpoint for PIVX data, optimised for least bandwidth
- * - No localisation, tickers, community data, developer data or sparklines
- */
-export const COINGECKO_ENDPOINT =
-    'https://api.coingecko.com/api/v3/coins/pivx?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false';
-
-/**
```
Comment (Duddino) [inline scripts/prices.js:11]: ​

File: webpack.dev.js [hunk 4]
```diff
@@ -1,18 +0,0 @@
...
-
-module.exports = merge(common, {
-    mode: 'development',
-    devServer: {
-        static: {
-          directory: path.join(__dirname, './'),
-        },
-        compress: true,
-        port: 5500,
-        hot: true,
```
Comment (Duddino) [inline webpack.dev.js:13]: ​

File: LICENSE [hunk 5]
```diff
@@ -1,23 +0,0 @@
-MIT License
```
Comment (Duddino) [inline LICENSE:1]: ​

File: .eslintrc.js [hunk 6]
```diff
@@ -1,28 +0,0 @@
-module.exports = {
```
Comment (Duddino) [inline .eslintrc.js:1]: ​

File: scripts/vanitygen_worker.js [hunk 7]
```diff
@@ -1,12 +0,0 @@
-import { deriveAddress } from './wallet.js';
-import { getSafeRand } from './misc.js';
-
-onmessage = function (_evt) {
```
Comment (Duddino) [inline scripts/vanitygen_worker.js:4]: ​

File: scripts/utils.js [hunk 8]
```diff
@@ -1,17 +0,0 @@
-import { Buffer } from 'buffer';
-import { sha256 } from '@noble/hashes/sha256';
-
-export function hexToBytes(str) {
-    return Buffer.from(str, 'hex');
-}
-
-export function bytesToHex(bytes) {
-    return Buffer.from(bytes).toString('hex');
```
Comment (Duddino) [inline scripts/utils.js:9]: ​

**Review by Duddino — APPROVED** [review]
​

**Review by Liquid369 — APPROVED** [review]
tACK 100b3302781d32d70d566860543e44757cb0be77

Absolutely agree with this merge!
LGTM.

🥇 🏆

**Comment by JSKitty** [comment]
Due to the nature of this PR, the merge commit is also zero-knowledge, meaning there is no commit.

**Pull request closed without merging**